src/benchmark/benchmark_lighting.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+Minetest
+Copyright (C) 2022 Minetest Authors
```

Comment:
if you create a wholly new file its okay to add your name here btw

Reply:
I left this here because it was the copyright notice in `benchmark_serialize.cpp` where I copied this file from.